pcs/rule.py

```diff
@@ -22,7 +22,7 @@ def parse_argv(argv, extra_options=None):
     """
     Commandline options: no options
     """
-    options = {"id": None, "role": None, "score": None, "score-attribute": None}
+    options = {"id": None, "role": None, "node":None, "score": None, "score-attribute": None}
```

Comment:
I'm not sure what is the point of this change. As explained in the other comment, there is no `node` option in location constraints.